pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -398,6 +398,10 @@ func createNodeCRD(clientset apiextensionsclient.Interface) error {
 											Type:        "string",
 											Description: "instance-id is the Azure specific identifier of the node",
```

Comment:
I think this `instance-id` field being here is outdated since it has been pulled out in the main spec part via https://github.com/cilium/cilium/commit/948440d4f192f75c0aac1c028536cee31c1a17a1.

Is there any thing we can do here related to validation to keep the CRD validation in-line with the actual structs?